docs/tutorials/build.mdx

```diff
@@ -1,8 +1,15 @@
 ---
-title: 'Building with xorq'
+title: 'Building and Running Expressions'
 ---
 
-xorq provides a command-line interface with a `build` command that allows you to generate artifacts from Ibis expressions defined in Python scripts. This tutorial will walk you through using this functionality.
+The xorq CLI provides two powerful commands, `build` and `run`, that help you separate the definition of data transformations from their execution. This separation enables:
+
+- **Reproducibility**: Define transformations once and run them consistently across different environments
```

Comment:
Reproducibity and Repliability sounds very close. 

Also Deployment and Version Control is the same thing. It seems repeated. Not sure if decoupling applies.